src/AirshipBindings.Android.Core/Additions/UAirship.cs

```diff
@@ -47,6 +47,10 @@ public AirshipReadyCallback(Action<UAirship> callback)
 			}
 
 			public void OnAirshipReady (UAirship airship) {
+				Version version = Assembly.GetExecutingAssembly().GetName().Version;
```

Comment:
This only works if they call takeOff or shared on UAirship directly. I think you want to have:

static UAirship {
  UAirship.shared( airship -> {
     airship.Analytics.RegisterSDKExttension(...);
   });
}